offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/DependencyManager.groovy

```diff
@@ -562,73 +518,63 @@ class DependencyManager {
     }
 
     /**
-     *
-     * @param sharedViewModel
-     * @param affiliationResourcesService
-     * @param customerResourcesService
-     * @param projectManagerResourcesService
-     * @param personResourcesService
-     * @param createAffiliationDataSource
-     * @param createPersonDataSource
+     * Creates a new SearchPersonView using fields
+     * <ul>
+     *     <li>{@link #personResourceService}</li>
+     * </ul>
+     * and methods
+     * <ul>
+     *     <li>{@link #createUpdatePersonView()}</li>
+     * </ul>
      * @return
      */
-    private static SearchPersonView createSearchPersonView(AppViewModel sharedViewModel,
-                                                           ResourcesService<Affiliation> affiliationResourcesService,
-                                                           ResourcesService<Customer> customerResourcesService,
-                                                           ResourcesService<ProjectManager> projectManagerResourcesService,
-                                                           ResourcesService<Person> personResourcesService,
-                                                           CreateAffiliationDataSource createAffiliationDataSource,
-                                                           CreatePersonDataSource createPersonDataSource) {
+    private SearchPersonView createSearchPersonView() {
+
+        ResourcesService<Person> personResourcesService = this.personResourceService
+
         // this event emitter is used to communicate between the search person view and the
-        // update person view
+        // update person view. The SearchPersonView emits persons to be updated. They are consumed
+        // by the UpdatePersonView
         EventEmitter<Person> personSelectEvent = new EventEmitter<Person>()
```

Comment:
is this replacing the event emitter from above?

Reply:
yes exactly